Automattic/woocommerce-payments
# Update deposit overview details

Fixes #1678 

#### Changes proposed in this Pull Request

* Move WC Summary component style overrides from overview to main `style.css`. We will be using the component in more, so makes sense to apply our custom style modifications globally.
* Update deposit detail to add more overview data:
  * Differentiate instant deposits using the text `Instant deposit date:` rather than `Deposit date`.
  * Added fee percentage and amount (in red if greater than zero).
  * Added net amount (in green)
*  Create a custom `SummaryNumber` component to avoid recreating the same styles. And allow the use of `className` for the `value` prop.

#### Testing instructions

* Access **Payments > Deposits** and click any deposit to see the details.
* Deposit overview should show:
  * If is an instant deposit in `Deposit date:` or `Instant deposit date:`
  * Deposit amount
  * Deposit fee percentage and amount
  * Deposit net amount
* You can edit `client/deposits/details/index.js` to alter the fees and see how deposit fee amount had red color when is different from zero.

### Screenshots

<img width="1090" alt="Screenshot 2021-05-18 at 15 49 46" src="https://user-images.githubusercontent.com/7670276/118662954-b2849700-b7f0-11eb-834d-924e17d4395a.png">

<img width="1092" alt="Screenshot 2021-05-18 at 14 52 51" src="https://user-images.githubusercontent.com/7670276/118662774-8ff27e00-b7f0-11eb-9d8a-cdc9fb95c136.png">


-------------------
